graphql/src/main/java/io/jmix/graphql/schema/FilterManagerImpl.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.graphql.schema;
+
+import com.google.common.collect.ImmutableList;
+import graphql.schema.GraphQLScalarType;
+import org.springframework.stereotype.Component;
+
+import java.util.EnumSet;
+import java.util.List;
+
+import static graphql.Scalars.*;
+import static io.jmix.graphql.schema.Types.FilterOperation.*;
+import static io.jmix.graphql.schema.scalar.CustomScalars.GraphQLBigDecimal;
+import static io.jmix.graphql.schema.scalar.CustomScalars.GraphQLLong;
+import static io.jmix.graphql.schema.scalar.CustomScalars.*;
+
+@Component("graphql_FilterManager")
```

Comment:
Short module name should be used in component name
`@Component("gql_FilterManager")`